blog-preview-card/styles.css

```diff
@@ -0,0 +1,107 @@
+body {
+  background-color: hsl(47, 88%, 63%);
+  font-family: "Figtree";
+  font-weight: 500;
+  margin: 0;
+}
+
+body p {
+  font-size: 18px;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 390px;
+  background-color: hsl(0, 0%, 100%);
+  border: 1px solid hsl(0, 0%, 7%);
+  border-radius: 20px;
+  box-shadow: 8px 8px 0px -3px hsl(0, 0%, 7%);
```

Comment:
The card shadow when not hovered is a little bit ticker on designs, but overall it's close enough

Reply:
🤔 
<img width="1201" alt="ima🤔 c="https://github.com/wojdanowski/exercises/assets/52794342/58b40b97-e805-4cf2-83b5-21fa8d2dc16b">